cypress/integration/articleFlows/removeMainImage.spec.js

```diff
@@ -0,0 +1,29 @@
+describe('Remove main image v2 editor', () => {
+  beforeEach(() => {
+    cy.testSetup();
+    cy.fixture('users/articleEditorV2User.json').as('user');
+
+    cy.get('@user').then((user) => {
+      cy.loginUser(user).then(() => {
+        cy.createArticle({
+          title: 'Test Article',
+          tags: ['beginner', 'ruby', 'go'],
+          content: `This is a test article's contents.`,
+          published: true,
+          mainImage:
+            'https://emojipedia-us.s3.dualstack.us-west-1.amazonaws.com/thumbs/240/twitter/248/dog-face_1f436.png',
+        }).then((response) => {
+          cy.visit(`${response.body.current_state_path}/edit`);
+        });
+      });
+    });
+  });
+
+  describe("Removing the article's main/cover image", () => {
+    it('should successfully remove the image', () => {
+      cy.findByText('Remove').click();
+      cy.findByText('Save changes').click();
+      cy.get('.crayons-article__cover').should('not.exist');
```

Comment:
Not sure if `.get(...).should('not.exist')` is the best assertion here, let me know if you think there's a better way. Here's the relevant HTML:
```erb
<header class="crayons-article__header" id="main-title">
  <% if @article.video.present? %>
    <%= render "articles/video_player", meta_tags: true, article: @article %>
      <% elsif @article.main_image.present? %>
        <div class="crayons-article__cover">
          <img src="<%= cloud_cover_url(@article.main_image) %>"
               width="1000"
               height="420"
               style="background-color:<%= @article.main_image_background_hex_color %>;"
               class="crayons-article__cover__image"
               alt="Cover image for <%= sanitize_and_decode @article.title %>">
        </div>
   <%# rest of the code %>
```

Reply:
Never mind I change it to `.findByAltText`.